docs/network_firewall.rst

```diff
@@ -465,20 +469,38 @@ to add a rule.
 Once you've set up the firewall, exit the Unsafe Browser, and continue
 with the "Keeping pfSense up to date" section below.
 
-Configuration Reference Templates
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+SecureDrop Configuration (Templates)
+------------------------------------
+
+Alternatively, you can load the provided ``.xml`` templates.
+
+First navigate to **Diagnostics ▸ Backup & Restore**:
 
 As an alternative to the provided screenshots, you can examine the provided
-``.xml`` templates as a reference:
+``.xml`` templates as a reference. However, because of the Tails chroot,
```

Comment:
This "templates as a reference" text should be removed, since these templates are no longer to be used as reference.